cibuildwheel/macos.py

```diff
@@ -121,12 +122,23 @@ def call(args, env=None, cwd=None, shell=False):
             call(['delocate-wheel', '-w', '/tmp/delocated_wheel', built_wheel], env=env)
         delocated_wheel = glob('/tmp/delocated_wheel/*.whl')[0]
 
+        test_drivers = []
         if test_command:
+            if 'intel' in config.identifier and config.version[0] == '3':
+                test_drivers = ['python', 'python-32']
+            else:
+                test_drivers = ['python']
+
+        for test_driver in test_drivers:
```

Comment:
Hmmm, nicely compact, this way, but now I'm wondering if, logically/for readability, it wouldn't make sense to have all of this inside the `if test_command:`?

Reply:
So just indent the for loop one level right then ?
You're probably right. I took a bad habit of finding tricks in order not to get too much multi-line statements with linters enforced rules (as the one coming in #212)

Reply:
I don't know, either. Too much indentation is really annoying either, especially with line length limits (not a big fan there, but fair enough, there's a standard for some reason?), but in this case ... up to your judgment, but worth a mention.

Reply:
Extract whole test to another function?